build/android/BUILD.gn

```diff
@@ -30,6 +30,19 @@ action("sign_app") {
   ]
 }
 
+action("collect_objs_app") {
+  script = "//brave/build/android/collect_obj_files.py"
```

Comment:
you don't need a python script for this, there is already a wrapper for calling commands like this `script = "//build/gn_run_binary.py"`

Reply:
we removed that part